test/quicer_SUITE.erl

```diff
@@ -3081,6 +3082,25 @@ tc_stream_get_owner_remote(Config) ->
     SPid ! done,
     ensure_server_exit_normal(Ref).
 
+tc_setopt_global_lb_mode_ifip(_Config) ->
+    {ok, _} = application:ensure_all_started(quicer),
+    true = code:soft_purge(quicer_nif),
+    true = code:delete(quicer_nif),
+    %% If test fail ensure we have this netdev
+    NetDevName =
+        case os:type() of
+            {unix, darwin} -> "lo0";
+            _ -> "lo"
+        end,
+    application:set_env(quicer, lb_mode, NetDevName),
+    quicer:reg_close(),
+    quicer:close_lib(),
+    {ok, _} = quicer:open_lib(),
+    ?assertEqual(
+        {ok, 2},
```

Comment:
At first, this looks like it could be:

```suggestion
        {ok, ?QUIC_LOAD_BALANCING_SERVER_ID_FIXED},
```

But, if we set `lb_mode` to `lo`, shouldn't it be a different number, like `16777343`?

Reply:
16777343 won't be a values supported by msquic, but quicer NIF use the magic to covert it to QUIC_LOAD_BALANCING_SERVER_ID_FIXED as the value of  `LoadBalancingMode` and at the same time set the `FixedServerID` to 16777343 which is a hint for custmized  LB.  

"lo" is just for testing. In real it should be the net dev that recv from the LB.

Reply:
I need to fix the release script as well, will update in that PR.